src/main/java/com/aws/greengrass/deployment/DefaultDeploymentTask.java

```diff
@@ -84,6 +91,18 @@ public DeploymentResult call() throws InterruptedException {
                     .kv("Deployment service config", deploymentServiceConfig.toPOJO().toString())
                     .log("Starting deployment task");
 
+            // download configuration if large
+            List<String> requiredCapabilities = deploymentDocument.getRequiredCapabilities();
+            if (requiredCapabilities != null && requiredCapabilities.contains(LARGE_CONFIGURATION)) {
```

Comment:
this feels weird. Can we check to see if the deployment requires downloading instead of checking the claimed required capability?

Reply:
I raised the same thing to cloud. Right now "Require Large Configuration" means "Download". I guess they are not 100% equivalent but not a big problem